modules/jpa/pom.xml

```diff
@@ -19,6 +19,18 @@
       <groupId>io.oasp.java.modules</groupId>
       <artifactId>oasp4j-jpa-dao</artifactId>
     </dependency>
+    <dependency>
+     <groupId>io.oasp.java.modules</groupId>
+  <artifactId>oasp4j-jpa-envers</artifactId>
+      </dependency>
+   <dependency>
+      <groupId>org.hibernate</groupId>
+      <artifactId>hibernate-envers</artifactId>
```

Comment:
Aint this a transitive dependency of `oasp4j-jpa-envers`?

Reply:
done